tasks/main.yml

```diff
@@ -59,11 +59,23 @@
   with_items: "{{ os_images_list }}"
   become: True
 
-- name: Download the image to the directory when an image_url is defined
-  get_url:
-    url: "{{ item.image_url }}"
-    dest: "{{ os_images_cache }}/{{ item.name }}/{{ item.name }}.{{ item.type | default('qcow2') }}"
-  with_items: "{{ os_images_list }}"
+- block:
+    - name: Download the image to the directory when an image_url is defined
+      get_url:
+        url: "{{ item.image_url }}"
+        dest: "{{ image_dest }}"
+      with_items: "{{ os_images_list }}"
+    - debug: var=image_dest
+      with_items: "{{ os_images_list }}"
+    - debug: var=image_is_xz
```

Comment:
Did you mean to leave the debug statements?

Reply:
No sir, definitely not, thanks!